doc/man1/openssl-cmp.pod.in

```diff
@@ -0,0 +1,1142 @@
+        -- value 7 is not used
+        removeFromCRL           (8),
+        privilegeWithdrawn      (9),
+        aACompromise           (10)
+    }
+
+=back
+
+
+=head2 Credentials format options
+
+=over 4
+
+=item B<-ownform> I<PEM|DER|P12>
+
+Format to try first when reading files and to use for writing files containing
+own (i.e., client-side) certificates.
+Default value is PEM.
+
+=item B<-keyform> I<PEM|DER|P12>
+
+Format to try first when reading key files.
+Default value is PEM.
+
+=item B<-otherform> I<PEM|DER|P12>
+
+Format to try first when reading files and to use for writing files containing
+certificates of other parties (i.e., the server side).
+Default value is PEM.
+
+=item B<-otherpass> I<arg>
+
+Pass phrase source for certificate given with the B<-trusted>, B<-untrusted>,
+B<-out_trusted>, B<-extracerts>, B<-tls_extra>, or B<-tls_trusted> options.
+If not given here, the password will be prompted for if needed.
+
+For more information about the format of B<arg> see the
+B<PASS PHRASE ARGUMENTS> section in L<openssl(1)>.
+
+=item B<-engine> I<id>
+
+Specifying a crypto engine B<id> will lead to obtaining a functional
+reference to the specified engine, initializing it if needed.
+The engine will be used for all algorithms supported for keys
+prefixed by C<engine:>.
+Engines may be defined in the OpenSSL config file as usual in an engine section.
+
+Options specifying keys, like B<-key>, B<-newkey>, B<-tls_key> can prefix
+C<engine:> to engine-specific identifiers for security tokens objects held by
+the engine.
+ The following example utilizes the RFC 7512 PKCS #11 URI scheme
+as supported, e.g., by libp11:
+C<-key engine:pkcs11:object=my-private-key;type=private;pin-value=1234>
+
+=back
+
+
+=head2 TLS options
+
+=over 4
+
+=item B<-tls_used>
+
+Enable using TLS (even when other TLS_related options are not set)
+when connecting to CMP server.
+
+=item B<-tls_cert> I<filename>
+
+Client's TLS certificate.
+If the file includes further certificates,
+they are used for constructing the client cert chain provided to the TLS server.
+
+=item B<-tls_key> I<filename>
+
+Private key for the client's TLS certificate.
+
+=item B<-tls_keypass> I<arg>
+
+Pass phrase source for client's private TLS key B<tls_key>.
+Also used for B<-tls_cert> in case it is an encrypted PKCS#12 file.
+If not given here, the password will be prompted for if needed.
+
+For more information about the format of B<arg> see the
+B<PASS PHRASE ARGUMENTS> section in L<openssl(1)>.
+
+=item B<-tls_extra> I<filenames>
+
+Extra certificates to provide to TLS server during TLS handshake
+
+=item B<-tls_trusted> I<filenames>
+
+Trusted certificate(s) to use for verifying the TLS server certificate.
+This implies hostname validation.
+
+Multiple filenames may be given, separated by commas and/or whitespace.
+Each source may contain multiple certificates.
+
+=item B<-tls_host> I<name>
+
+Address to be checked (rather than B<-server> address) during hostname
```

Comment:
Is this mandatory - or is it derived somehow if not given?

Reply:
I've made clear that it is optional, defaulting to the server address.